[IMP] sales: pricelist update to 19.4 (#19025)
odoo/documentation: content/applications/sales/sales/products_prices/prices/pricing.rst

Reviewer: This can be:

Use case: Apply a discount and round to `0.99`

There's a better way to write that but I am coming up short on ideas! =D

Then you can intrpduce th euse case:
Retailers often have limited time promotions, whre percentage discounts are applied ot products. If a retailer wants to have al prices end is a specifi amount, such as `9.99`, the price list is configured as follows:

(list items to configure)

Reviewer: ```suggestion
Customer specific pricelists
```
Maybe?

Reviewer: ```suggestion
While the default pricelist applied to any customer is the :guilabel:`Public Pricelist`, Odoo
provides the opportunity to directly apply a different pricelist to customers. This is done on the cusotmer's contact form.
```

Reviewer: ```suggestion
To set a specific pricelist for a customer, open the desired customer's contact form, either by navigating to :menuselection:`Sales
```
Just to claify

Reviewer: ```suggestion
In the *Sales & Purchase* tab of the desired customer's contact form, designate which pricelist should be applied using the drop-down menu in the :guilabel:`Pricelist` field.
```
I think this is more clear, less wordy

Reviewer: I know you didn't take this, but there are no more red arorows/markups on images- so I'd replace this if you could

Reviewer: This is so small to see- I'd replace these if you could, zoomed in more, and set to a width of 933

Author: I decided to remove the form iamge as the main focus is the tab not the Customer form it's self.

Odoo **Sales** has a useful pricelist feature that can be tailored to fit any unique pricing
strategy. Pricelists suggest certain prices, but they can always be overridden on the sales order.

.. seealso::
   - :doc:`/applications/sales/sales/products_prices/prices/currencies`
   - :doc:`/applications/websites/ecommerce/configuration/prices`
   - :doc:`../../../subscriptions`

.. _sales/products/pricelist-configuration:

Configuration
=============

To enable pricelists in the Odoo **Sales** app, first navigate to :menuselection:`Sales app -->

> Configuration --> Settings`. In the :guilabel:`Pricing` section, enable the checkbox next to the

Reviewer: ```suggestion
Configuration --> Settings`. In the *Pricing* section, enable the checkbox next to the
```
Sections and tabs are italicized

:guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.

> .. image:: pricing/pricelist-feature-setting.png

Reviewer: I think thi can be removed- not a value add, and so basic no one would have a problem finding this

Author: I received feedback before that readers like seeing the feature image, even if it's easy to find via the text. So i just leave it in.

   :alt: How the pricelist feature setting looks in Odoo Sales.

Access the *Pricelists* page by either selecting the :icon:`oi-arrow-right` :guilabel:`Pricelists`

> link (beneath the :guilabel:`Pricelists` feature on the *Settings* page), or navigating to

Reviewer: ```suggestion
Access the *Pricelists* page by either clicking the :icon:`oi-arrow-right` :guilabel:`Pricelists`
link (beneath the :guilabel:`Pricelists` feature on the *Settings* page), or navigating to
```
I think that's more accurate since you click it, not select from a variety of options.

:menuselection:`Sales app --> Products --> Pricelists`.

Either option reveals the *Pricelists* page, in which pricelists can be created and modified at any
time.

.. important::
   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`
   pricelist is applied. The :guilabel:`Default` pricelist is the first pricelist found in
   :menuselection:`Sales app --> Products --> Pricelists` without an assigned :guilabel:`Country
   Group`. Odoo reads the :guilabel:`Pricelist Name` column from top to bottom to determine the
   :guilabel:`Default` pricelist.

.. note::
   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows
   website visitors to choose a pricelist when shopping in the **eCommerce** website.

> .. image:: pricing/18-sales-pricelist-page.png

Reviewer: This is super small and hard to read- I recommend making the columns thinner so it's less wide. and you can make images up to 933 pixels wide (not the 8-something) This I learned from Sam, and I have always used this for ones with lots of text that need to be larger. When you make the columns shorter, and they are not as wide as the search bar, I crop out the title and search bar- only show the actual list, from the headings, etc. That also helps.

   :alt: How the pricelists page looks in Odoo Sales.

.. _sales/products/create-edit-pricelists:

Creating and editing pricelists
-------------------------------

From the *Pricelists* page, either select the pricelist to edit, or click :guilabel:`New` to create
a new pricelist, which reveals a blank pricelist form that can be configured in a number of

> different ways.

Reviewer: ```suggestion
From the *Pricelists* page, either select the pricelist to edit or click :guilabel:`New` to create
a new pricelist, which reveals a blank pricelist form that can be configured to suit the business's needs.
```
'A variety of ways' sounded a bit vague - I would have recommended listing the ways, but I think this is a good workaround so it's less vague and more descriptive.

When creating a new pricelist, start by adding a name for the pricelist in the blank field at the

> top of the form. Next, select which :guilabel:`Currency` should be used.

Reviewer: This is a personal preference of my own, but I always explicitly state the method used - 'using the drop-down menu, ' 'type in the field, ' 'click the option,' etc. You don't have to, but I think it adds another layer of information that someone may need. If they can follow the easy steps, then the complex ones are a bit easier for them to understand in more complex workflows.

If working in a multi-company environment, select which company this pricelist should apply to in
the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically applied
to all companies in the database.

> If working in a multinational company, select the countries where this pricelist will apply under

Reviewer: ```suggestion
If working in a multinational company, select the countries where this pricelist applies to under
```

the :guilabel:`Country Groups` field.

> .. image:: pricing/19-4-sales-new-pricelist-form.png

Reviewer: I would fully have this fille dout - add the title, the ocuntry gorups, etc - everything you mentioned above should be visible in the form (we shouldn't have blank forms or partially-filled out forms)

   :alt: How the pricelist detail form looks in Odoo Sales.

.. _sales/products/price-rules:

Adding price rules
------------------

In the :guilabel:`Rules` tab, each line creates a new record that implements customized pricing to
the sales order where the pricelist is applied. This can be used to create complex pricing

> structures, such as progressive discounts when greater quantities of a product are purchased.

Reviewer: ```suggestion
In the *Rules* tab, each line creates specific pricing rules. When the pricelist is used on a sales order, Odoo applies those rules to calculate the prices.This can be used to create complex pricing
structures, such as progressive discounts when greater quantities of a product are purchased.
```
This sounds more clear to me- I was getting lost in that first sentence =)

> To create a new price rule, click on :guilabel:`Add a line`, which opens the *Create Pricelist

Reviewer: ```suggestion
To create a new price rule, click :guilabel:`Add a line`, which opens a *Create Pricelist
```
I say 'click on' a lot and it alwasy gets edited ot just 'click' so I am trying to do that from now on.

Rules* pop-up form.

In the :guilabel:`Apply To` field, select whether the price rule applies to a :guilabel:`Product` or
:guilabel:`Category`. Depending on the selection, the next field is either :guilabel:`Product` or

> :guilabel:`Category`. Specify a product or category the pricelist applies to or leave the default

Reviewer: ```suggestion
In the :guilabel:`Apply To` field, select whether the price rule applies to a specific :guilabel:`Product` or a product
:guilabel:`Category`. Depending on the selection, the next field is either :guilabel:`Product` or
:guilabel:`Category`. Specify the product or category the pricelist applies to, or leave the field blank to apply to all products or categories.

> selection, :guilabel:`All products` or :guilabel:`All categories` respectively.

Reviewer: ```suggestion
```

Question - is there a circumstance where you'dleave that field blank for categpry? I see selecting product, leaving product blank, and it applies a blanket disocunt ot everything  0 but the same woud work for the category (seems like an odd thing ot do). Basically, it seems that leaving the field blank applies to app products, regardless of category, no katter the selection. if that is so, you can add a note about that- that it does the same thing? Just a suggestion - and I don't honestly know the asnwer.

For :guilabel:`Price Type`, select if the specialized pricing falls under one of the following:
:ref:`Discount <sales/pricing/discount>`, :ref:`Formula <sales/pricing/formula>`, or :ref:`Fixed
Price <sales/pricing/fixed-price>`. Refer to the :ref:`sales/pricing/price-types` section for

> additional configuration instructions on how to set up the price types.

Reviewer: ```suggestion
For :guilabel:`Price Type`, select how the prices are adjusted, either applying a :ref:`Discount <sales/pricing/discount>`, using a specific :ref:`Formula <sales/pricing/formula>`, or  applying a  :ref:`Fixed
Price <sales/pricing/fixed-price>`. 
```
Since we link ot the sections, and it's clear that they're links, we do'n tneed the second sentence after it. I also made the sentence more descriptive so they can more clearly understand what they mean (shoul dbe obvious but you never know).

.. note::
   If a price rule is set for a particular product, and another one for its product category, Odoo

>    takes the rule of the product itself.

Reviewer: ```suggestion
.. important::
   If a price rule is set for a particular product, and another one for its product category, Odoo
   prioritizes the *product* rule and applies it.
```
This seems important - not just a tip

> If the price rule is part of an subscription, in the :guilabel:`Recurring Plan` field drop-down menu

Reviewer: ```suggestion
If the price rule is part of a subscription, in the :guilabel:`Recurring Plan` field drop-down menu
```

select a recurrence period (e.g., Monthly, Quarterly, Weekly, etc.). New recurrence periods can also

> be created from this field by clicking :guilabel:`Search more` and clicking :guilabel:`Create New`.

Reviewer: ```suggestion
select a recurrence period. The default selections are :guilabel:`Monthly`,:guilabel:`6 Months`, or :guilabel:`Weekly`. New recurrence periods can also
be created from this field by clicking :guilabel:`Search more` and clicking :guilabel:`Create New`.
```
Verify if this is true in a blank 19.4 database trial - but in Runbot (which is never reliable, and why I always make a blank one to test the defaults) I only see 3 default selections.

Reviewer: Also - is there documentation on making these time periods in the subscriptions app? If so, you can link the words 'create a new recurring plan...' to the doc (then you don't need to say 'click Create New' etc.)

> Refer to the Odoo :doc:`/applications/sales/subscriptions` documentation for more information.

Reviewer: Could be a 'see also' instead, at the end of this section.

If the discount applies based off the number of products ordered, enter the minimum quantity of

> selected products in the :guilabel:`Min Qty` for this pricelist to apply.

Reviewer: ```suggestion
If the discount is only applied when a specific number of a product is ordered, enter the amount in the :guilabel:`Min Qty` field.
```

The :guilabel:`Packaging` field has no name and is next to the :guilabel:`Min Qty` field. This field
defaults to the assigned :abbr:`UoM (Unit of Measure)` configured on the selected product. This

> field only displays when the price rule is configured for *Products*.

Reviewer: ```suggestion
The packaging field appers next to the :guilabel:`Min Qty` field, but is not labeled. This field
defaults to the assigned :abbr:`UoM (Unit of Measure)` configured on the selected product. This
field only displays when the price rule is configured for *Products*.
```

Reviewer: Not sure if 'packaging' should be italicized- only because it doe snot appear anywhere

If the discount is only valid on specific dates, enter the start and end date in the

> :guilabel:`Validity Period` field during which this pricelist can be applied to quotations.

Reviewer: ```suggestion
If the discount is only valid on specific dates, such as a limited time sale, enter the start and end date in the
:guilabel:`Validity Period` field during which this pricelist can be applied to quotations.
```
totally optional, just to add some more ocntext/example

Once all configurations are complete, either click :guilabel:`Save & Close` to save the pricelist
rule, or click :guilabel:`Save & New` to immediately create another pricelist rule on a fresh form.

.. _sales/pricing/price-types:

Configuring price types
-----------------------

The *Create Pricelist Rules* pop-up form offers three different price types to offer discounts or
price increases.

- :ref:`Discount <sales/pricing/discount>`
- :ref:`Formula <sales/pricing/formula>`

> - :ref:`Fixed Price <sales/pricing/fixed-price>`

Reviewer: Since these are so short, you can add a colon and small explanation if you want :

Discount: Use this to apply a specific discount to all products or a product category.

.. _sales/pricing/discount:

Configuring a discount
~~~~~~~~~~~~~~~~~~~~~~

> .. important::

Reviewer: Just a preference - I tent to avoid starting any section wih an admonishion block, and put it either after the instructions it relates to, or beneath the images. Not required, just a preference of mine.

   If a pricelist is applied to a sales quotation with the :guilabel:`Price Type` set as
   :guilabel:`Discount`, the discount is visible to the customer.

To figure a discount using percentage as the computation, select :guilabel:`Discount` for
:guilabel:`Price Type` and enter the percentage to be discounted. A mark-up can be configured by
using a negative value in this field.

Next select the base price Odoo uses to calculate with the discount percentage. The default is
:guilabel:`sales price` which refers to the *Sales price* field on the product form. Other
pricelists can be selected for the base price from the drop-down menu. New pricelists can also be
created from this menu by clicking :guilabel:`Search more` and clicking :guilabel:`Create New`.

.. image:: pricing/sales-pricelist-pricerule-discount.png
   :alt: A pricelist rule using Price Type: Discount to create a discount.

.. image:: pricing/sales-pricelist-pricerule-discount-customer.png
   :alt: How a pricelist rule using Price Type: Discount appears on customer preview.

> 

Reviewer: OK, after thinking, I think it might be a god idea to move the important block to down here, then this can be inside that block- since it is illustrating the issue in the inportant block. Then we won't have 2 images stacked on top of each other (very rare)

.. _sales/pricing/formula:

Configuring a formula
~~~~~~~~~~~~~~~~~~~~~

.. important::
   If a pricelist is applied to a sales quotation with the discount :guilabel:`Price Type` set as
   :guilabel:`Formula`, the discount will *not* be visible to the customer.

The :guilabel:`Formula` option computes the pricelist rules based the following configuration:

> - :guilabel:`Base price`: Select either the :guilabel:`Sales price`, :guilabel:`Cost Price`, or

Reviewer: ```suggestion
- :guilabel:`Based price`: Select either the :guilabel:`Sales price`, :guilabel:`Cost`, or
```
I hate the grammar of this, but it's how it appears on the UI

  :guilabel:`Other Pricelist` to use as the base price for the computation. The :guilabel:`Sales
  Price` and :guilabel:`Cost Price` are based on the same fields on the product form. The

>   :guilabel:`Other Pricelist` uses a pricelist as its base price.

Reviewer: ```suggestion
  :guilabel:`Other Pricelist` to use as the base price for the computation. The :guilabel:`Sales
  Price` and :guilabel:`Cost` are based on the corresponding fields on the product form. The
  :guilabel:`Other Pricelist` uses a pricelist as its base price.
```

- :guilabel:`Discount`: Percentage discount to be applied. Negative values can be entered to

>   increase prices.

Reviewer: ```suggestion
- :guilabel:`Discount`: The applied discount percentage. Negative values can be entered to
  increase prices.
```

> - :guilabel:`Round off to`: Numerical value to act as round-off multiple, to be applied after

Reviewer: Ill be honest, I have no idea what this first sentence is saying =D  I would rewrite this so it's a bit more clear. Is it:

If prices should always appear in a specific format, such as `9.99`, or `10.00`, enter the rounding amount in this field.

  discount. The rounding method sets the price so that it is a multiple of the value in this field.

   .. note::

>       Rounding is applied *after* the discount and *before* the extra fee.

Reviewer: ```suggestion
  discount. The rounding method sets the price so that it is a multiple of the value in this field. This is applied *after* the discount but *before* the extra fee.
```
Just becausw there are so many admonition blocks stacked on top of each othe rin this section, trying to remove/move a few of them.

- :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
  :guilabel:`Round off to` have been applied.

.. tip::
   To have prices that end in 9.99, set the :guilabel:`Rounding Method` to `10` and the
   :guilabel:`Extra Fee` to `-0.01`.

> .. example::

Reviewer: Because both of these are large, and specific, I recommend these both become "Use Case" examples, and not in a block, but in it's own section within this section. So the title for this coul be:

Use case: Markup with a specific margin

Or whatever osunds best. This way, it can be expanded s you can explain everything further. I didn't know where the "margin" was supposed to go, since that word is not anywhere else in this doc. This way you can expand the beginning with the last sentence here:

Often in retail stores, prices are set based on a pspecific markup, and rwquire set margins to ensure profitability. To create a pricelist so products are priced at 200% of their cost, and have a set margin that products cannot be sold below, the pricelist rules are ocnfigured as follows:

(then you can list out the field, and why you select them. So :

- Base Price: Select 'Cost' to calculate pricing based on the cot of the goods.
- Markup: This field appears once 'Cost' is selecte din the 'Base Price' field. Enter `200` in this field to set the price to double the cost. 

etc.

time.

.. important::
   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`
   pricelist is applied. The :guilabel:`Default` pricelist is the first pricelist found in
   :menuselection:`Sales app --> Products --> Pricelists` without an assigned :guilabel:`Country
   Group`. Odoo reads the :guilabel:`Pricelist Name` column from top to bottom to determine the
   :guilabel:`Default` pricelist.

.. note::
   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows
   website visitors to choose a pricelist when shopping in the **eCommerce** website.

.. image:: pricing/19-4-sales-pricelist-page.png
   :alt: Example of the pricelists page in Odoo Sales.

.. _sales/products/create-edit-pricelists:

Creating and editing pricelists
-------------------------------

From the *Pricelists* page, either select the pricelist to edit or click :guilabel:`New` to create a
new pricelist, which reveals a blank pricelist form that can be configured to suit the business's
needs.

When creating a new pricelist, add the name for the pricelist by clicking in the blank field at the

> top of the form. Next, select the type :guilabel:`Currency` the pricelist uses by clicking in the

Reviewer 2: ```suggestion
top of the form. Next, select the type of :guilabel:`Currency` the pricelist uses by clicking in the
```

field to access the drop-down menu and selecting an option.

If working in a multi-company environment, select which company this pricelist should apply to in
the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically applied
to all companies in the database.

> If working in a multinational company, select the countries where this pricelist applies to under

Reviewer 2: ```suggestion
If working in a multinational company, select the countries where this pricelist applies under
```

the :guilabel:`Country Groups` field.

.. image:: pricing/19-4-sales-new-pricelist-form.png
   :alt: How the pricelist detail form looks in Odoo Sales.

.. _sales/products/price-rules:

Adding price rules
------------------

In the *Rules* tab, each line creates specific pricing rules. When the pricelist is used on a sales

> order, Odoo applies those rules to calculate the prices.This can be used to create complex pricing

Reviewer 2: ```suggestion
order, Odoo applies those rules to calculate the prices. This can be used to create complex pricing
```

structures, such as progressive discounts when greater quantities of a product are purchased.

To create a new price rule, click :guilabel:`Add a line`, which opens a *Create Pricelist Rules*
pop-up form.

In the :guilabel:`Apply To` field, select whether the price rule applies to a specific
:guilabel:`Product` or a product :guilabel:`Category`. Depending on the selection, the next field is
either :guilabel:`Product` or :guilabel:`Category`. Specify the product or category the pricelist
applies to, or leave the field blank to apply to all products or categories.

.. important::
   If a product doesn't have an assigned category then it is not included in the :guilabel:`All
   categories` option.

For :guilabel:`Price Type`, select how the prices are adjusted, either applying a :ref:`Discount

> <sales/pricing/discount>`, using a specific :ref:`Formula <sales/pricing/formula>`, or  applying a

Reviewer 2: ```suggestion
<sales/pricing/discount>`, using a specific :ref:`Formula <sales/pricing/formula>`, or applying a
```

:ref:`Fixed Price <sales/pricing/fixed-price>`.

.. important::
   If a price rule is set for a specific product, and another one for its product category, Odoo
   prioritizes the *product* rule and applies it.

> If the price rule is part of a subscription, in the :guilabel:`Recurring Plan` field drop-down menu

Reviewer 2: ```suggestion
If the price rule is part of a subscription, in the :guilabel:`Recurring Plan` field drop-down menu,
```

select a recurrence period. The default selections are :guilabel:`Monthly`, :guilabel:`6 Months`, or
:guilabel:`Weekly`. New recurrence periods can also be created from this field by clicking
:guilabel:`Search more` and clicking :guilabel:`Create New`.

If the discount is only applied when a specific number of a product is ordered, enter the amount in
the :guilabel:`Min Qty` field.

If a product has the *Packagings* field configured on its product form, users can create pricing for
individual packing options. The packaging field appears next to the :guilabel:`Min Qty` field, but
is not labeled. Clicking the field displays a drop-down menu of configured packaging options.

The field defaults to the assigned :abbr:`UoM (Unit of Measure)` configured on the selected product.
The packaging field only displays when the price rule is configured for *Products*.

> If the discount is only valid on specific dates, such as a limited time sale, enter the start and

Reviewer 2: ```suggestion
If the discount is only valid on specific dates, such as a limited-time sale, enter the start and
```

> end date in the :guilabel:`Validity Period` field during which this pricelist can be applied to

Reviewer 2: ```suggestion
end dates in the :guilabel:`Validity Period` field during which this pricelist can be applied to
```

quotations.

Once all configurations are complete, either click :guilabel:`Save & Close` to save the pricelist
rule, or click :guilabel:`Save & New` to immediately create another pricelist rule on a fresh form.

.. seealso::
   :doc:`/applications/sales/subscriptions`

.. _sales/pricing/price-types:

Configuring price types
-----------------------

The *Create Pricelist Rules* pop-up form offers three different price types to offer discounts or
price increases.

- :ref:`Discount <sales/pricing/discount>`: Applies a percentage discount to all products or a
  product category.
- :ref:`Formula <sales/pricing/formula>`: Applies a customizable discount using multiple values, for
  more control over the calculation.
- :ref:`Fixed Price <sales/pricing/fixed-price>`: Sets a flat sales price for all products or a
  product category.

.. _sales/pricing/discount:

Configuring a discount
~~~~~~~~~~~~~~~~~~~~~~

> To figure a discount using percentage as the computation, select :guilabel:`Discount` for

Reviewer 2: ```suggestion
To configure a discount using a percentage as the computation, select :guilabel:`Discount` for
```

:guilabel:`Price Type` and enter the percentage to be discounted. A mark-up can be configured by
using a negative value in this field.

> Next select the base price Odoo uses to calculate with the discount percentage. The default is

Reviewer 2: ```suggestion
Next, select the base price Odoo uses to calculate with the discount percentage. The default is
```

> :guilabel:`sales price` which refers to the *Sales price* field on the product form. Other

Reviewer 2: ```suggestion
:guilabel:`sales price`, which refers to the *Sales price* field on the product form. Other
```

pricelists can be selected for the base price from the drop-down menu. New pricelists can also be
created from this menu by clicking :guilabel:`Search more` and clicking :guilabel:`Create New`.

.. image:: pricing/sales-pricelist-pricerule-discount.png
   :alt: A pricelist rule using Price Type as Discount to create a discount.

.. important::
   If a pricelist is applied to a sales quotation with the :guilabel:`Price Type` set as
   :guilabel:`Discount`, the discount is visible to the customer.

.. _sales/pricing/formula:

Configuring a formula
~~~~~~~~~~~~~~~~~~~~~

.. important::
   If a pricelist is applied to a sales quotation with the discount :guilabel:`Price Type` set as
   :guilabel:`Formula`, the discount will *not* be visible to the customer.

The :guilabel:`Formula` option calculates pricelist rules using these settings:

> - :guilabel:`Based price`: Choose the base for the calculation — :guilabel:`Sales price`,

Reviewer 2: Is this supposed to be "based price" or "base price"?

Author: The gui for this is 'Based price'

  :guilabel:`Cost` (both pulled from the product form), or :guilabel:`Other Pricelist` (uses another
  pricelist as the base).
- :guilabel:`Discount`: The discount percentage to apply. Enter a negative number to increase the
  price instead.
- :guilabel:`Round off to`: Rounds the price to the nearest multiple of this value. Applied *after*
  the discount but *before* the :guilabel:`Extra Fee`.
- :guilabel:`Extra Fee`: Fixed amount added or subtracted after :guilabel:`Discount` and
  :guilabel:`Round off to` are applied.

.. tip::

>    To have prices that end in 9.99, set the :guilabel:`Rounding Method` to `10` and the

Reviewer 2: Should this be "Round off to" and not "Rounding Method"?

   :guilabel:`Extra Fee` to `-0.01`.

.. _sales/pricing/fixed-price:

Configuring a fixed price
~~~~~~~~~~~~~~~~~~~~~~~~~

To configure a :guilabel:`Fixed Price`, enter the new value that overrides the configured *Sales
Price* on a product form. The :guilabel:`Fixed Price` value is applied per unit. When applied on a

> quotation, the *Unit Price* for each order line on uses the :guilabel:`Fixed Price` value.

Reviewer 2: ```suggestion
quotation, the *Unit Price* for each order line uses the :guilabel:`Fixed Price` value.
```

pricelist rule, or click :guilabel:`Save & New` to immediately create another advanced pricelist
rule on a fresh form.

Pricelist rule logic
====================

Odoo assigns the top pricelist on the *Pricelist* page as default to a quotation as long as there is
no assigned Pricelist and the pricelist's values are true. This means based on the quotation form
information, Odoo applies the pricelist that best meets the quotation conditions starting from the
top of the *Pricelist* page.

When multiple pricelist rules apply to the same product on the same pricelist, Odoo determines
priority using this order:

1. :ref:`Most specific to least specific <sales/pricing/most-to-least-specific>`: More specific
   rules take priority over general ones.
#. :ref:`Minimum quantity (largest first) <sales/pricing/minimum-quantity>`: The higher quantity
   thresholds take priority over lower ones.
#. :ref:`Category order <sales/pricing/category-order>`: Manually configured category hierarchy
   breaks remaining ties where the subcategory takes priority over the parent category.
#. :ref:`Creation date (newest first) <sales/pricing/creation-date>`: The most recently created rule

>    takes priority is all else is equal.

Reviewer 2: ```suggestion
   takes priority if all else is equal.
```

.. _sales/pricing/most-to-least-specific:

1. Most specific to least specific
----------------------------------

A rule on a specific variant always takes priority over a broader rule, so a special price for one
variant overrides a general category discount.

1. Product variant: Applies to the selected variant of the product.
#. Product: Applies to all variants of a product.

> #. Subcategory: Applies to all products within the subcategory

Reviewer 2: ```suggestion
#. Subcategory: Applies to all products within the subcategory.
```

#. Parent category: Applies to all products in the parent category.
#. All products: Applies to all products within the database.
#. All categories: Applies to all products with an assigned category. This doesn't include
   uncategorized products.

.. _sales/pricing/minimum-quantity:

2. Minimum quantity (largest first)
-----------------------------------

> If the product specificity is the same then, rules with a higher minimum quantity take priority over

Reviewer 2: ```suggestion
If the product specificity is the same, then rules with a higher minimum quantity take priority over
```

rules with a lower (or no) minimum. This keeps bulk-pricing rules (e.g., 10 units) being overridden

> by a generic rule that also matches.

Reviewer 2: ```suggestion
rules with a lower (or no) minimum. This keeps bulk-pricing rules (e.g., 10 units) from being
overridden by a generic rule that also matches.
```

.. _sales/pricing/category-order:

3. Category order
-----------------

If rules have the same specificity and same minimum quantity, Odoo uses the manually configured

> category hierarchy. Meaning the sequence set on the categories themselves.

Reviewer 2: ```suggestion
category hierarchy, meaning the sequence is set on the categories themselves.
```

to override older ones with identical conditions, which is useful when rules are updated or replaced
without deleting the old one immediately.

.. _sales/pricing/customer-pricelist-application:

Customer specific pricelists
============================

When a customer is added to the database, the *Default* pricelist is automatically applied to the
*Pricelist* field on a contact form. Even if the field is left blank during :ref:`contact creation
<contacts/contact-form>`, the *Default* pricelist is automatically assigned when that contact form
is opened again.

To set a specific pricelist for a customer, go to :menuselection:`Sales app --> Orders -->
Customers` and select the customer from the *Customers* page, or click the customer's name on a
sales order.

On the customer's contact form, click the :guilabel:`Sales & Purchase` tab, in the *Sales* section,
select a pricelist using the :guilabel:`Pricelist` field drop-down menu.

.. image:: pricing/customer-form-pricelist-field.png
   :alt: The pricelist field in a customer detail form in Odoo Sales.

When the customer is added to a quotation, the *Pricelist* field is auto-populated based on the

> information from their contact form. User can manually change the Pricelist using the *Pricelist*

Reviewer 2: ```suggestion
information from their contact form. Users can manually change the Pricelist using the *Pricelist*
```

field on the quotation.

.. _sales/pricing/price-type-use-cases:

Price type use cases
====================

- :ref:`sales/pricing/discount-use-cases`
- :ref:`sales/pricing/formula-use-cases`
- :ref:`sales/pricing/fixed-price-use-cases`

.. _sales/pricing/discount-use-cases:

Discount use cases
------------------

The *Discount* price type is used when doing simple discount calculations on a product's sales price
or when using another pricelist as the base value.

Seasonal discount pricelist
~~~~~~~~~~~~~~~~~~~~~~~~~~~

A retail store holds a yearly summer sale. The sale is 25% off all summer dresses.

Navigate to :menuselection:`Sales app --> Products --> Pricelist` and select an existing pricelist
or click :guilabel:`New` to create a new one.

On the pricelist form, click :guilabel:`Add a line` to open the *Create Pricelist Rules* pop-up
window.

Configure the form using the following settings:

- :guilabel:`Apply To`: Select :guilabel:`Category`.

> - :guilabel:`Category`: Select *Summer dresses* from drop-down menu.

Reviewer 2: ```suggestion
- :guilabel:`Category`: Select *Summer dresses* from the drop-down menu.
```

- :guilabel:`Price Type`: Select :guilabel:`Discount`.
- :guilabel:`Discount`: Enter `25` and leave the default :guilabel:`sales price`.
- :guilabel:`Min Qty`: Leave at `0`.
- :guilabel:`Validity`: Select :guilabel:`Range` and set the date range to `July 1 - Aug 31`.

Click :guilabel:`Save & close` to add the pricelist rule to the pricelist.

.. image:: pricing/19-4-example-summer-dress-25-discount.png
   :alt: Example of 25% discount pricelist rule configuration.

Applying a 10% markup
~~~~~~~~~~~~~~~~~~~~~

A shoe store has increased sales targets for the year, and the company decided to add an additional
10% to the sales price of their rain boots. The wholesale price for the rain boots is $40.

Navigate to :menuselection:`Sales app --> Products --> Pricelist` and select an existing pricelist
or click :guilabel:`New` to create a new one.

On the *Create Pricelist Rules* pop-up window, follow these settings:

- :guilabel:`Apply To`: Select :guilabel:`Product`.

> - :guilabel:`Product`: Select *Rainboots* from the drop-down menu.

Reviewer 2: ```suggestion
- :guilabel:`Product`: Select *Rain boots* from the drop-down menu.
```

- :guilabel:`Price Type`: Select :guilabel:`Discount`.
- :guilabel:`Discount`: Enter `-10` and leave the default :guilabel:`sales price`.

Click :guilabel:`Save & close` to add the pricelist rule to the pricelist.

.. image:: pricing/19-4-example-rain-boots-discount.png
   :alt: Example of 10% markup pricelist rule configuration.

.. _sales/pricing/formula-use-cases:

Formula use cases
-----------------

The *Formula* price type is useful for complex pricing requirements, such as dynamic pricing values
based on margin cost, or pricing values that require both a discount and a flat rate to be added or
removed.

Dynamic pricing based on cost
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> It's approaching Winter break, and the most in-demand item is a jacket worn by a popular actress. A

Reviewer 2: ```suggestion
It's approaching winter break, and the most in-demand item is a jacket worn by a popular actress. A
```

clothing retail store stocks the jacket, but its inventory is running low. The store decides to sell
the jacket at a 100% markup with a $5 minimum margin during the winter season while the demand is
still high. The wholesale price of the jacket is $75 and the cost is $70.

First, navigate to :menuselection:`Sales app --> Products --> Pricelist` and select an existing
pricelist or click :guilabel:`New` to create a new one.

On the *Create Pricelist Rules* pop-up window, follow these settings:

- :guilabel:`Apply To`: Select :guilabel:`Product`.
- :guilabel:`Product`: Select *Classic Brown Jacket* from the drop-down menu.

> - :guilabel:`Price Type`: Select :guilabel:`Formula`

Reviewer 2: ```suggestion
- :guilabel:`Price Type`: Select :guilabel:`Formula`.
```

- :guilabel:`Base price`: Select :guilabel:`Cost` from the drop-down menu.
- :guilabel:`Markup`: Enter `100`.
- :guilabel:`Validity`: Select :guilabel:`Range` and select `Nov 1 to Dec 31`.

Click :guilabel:`Save & Close` to add the pricelist rule to the pricelist.

.. image:: pricing/19-4-example-jacket-formula.png
   :alt: Formula example for a markup cost with 5 dollar minimum margin in Odoo Sales.

Then create a quotation for the jacket, select desired pricelist from the :guilabel:`Pricelist`
drop-down menu and :ref:`set up the Margin and Margin (%) columns
<sales/margin/margin-sale-order-line>` to show what the values are.

.. image:: pricing/19-4-example-formula-markup-with-margins.png
   :alt: Example of a quotation with a pricelist applied and margins enabled.

.. seealso::
   :doc:`../../sales_quotations/margin`

20% discounts with prices rounded up to $9.99
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

A grocery store is having a flash sale on their chicken products in the butcher section. All the
chicken meat products are on sale for 20% and rounded to 9.99.

> Navigate to :menuselection:`Sales app > Products > Pricelist --> ` and select an existing pricelist

Reviewer 2: ```suggestion
Navigate to :menuselection:`Sales app --> Products --> Pricelist` and select an existing pricelist
```

Gotta be careful about this when copying and pasting. :)

or click :guilabel:`New` to create a new one.

On the *Create Pricelist Rules* pop-up window, follow these settings:

- :guilabel:`Apply To`: Select :guilabel:`Category`.
- :guilabel:`Category`: Select *Meat/Chicken* from the drop-down menu.
- :guilabel:`Price Type`: Select :guilabel:`Formula`.
- :guilabel:`Base price`: Select :guilabel:`Cost` from the drop-down menu.
- :guilabel:`Markup`: Enter `20`.
- :guilabel:`Round off to`: Enter `10`.
- :guilabel:`Extra Fee`: Enter `-0.01`.

Click :guilabel:`Save & Close` to add the pricelist rule to the pricelist.

.. image:: pricing/19-4-example-chicken-formula.png
   :alt: Example of a 20% discount with prices rounded to 9.99 in Odoo Sales.

.. _sales/pricing/fixed-price-use-cases:

Fixed price use cases
---------------------

The *Fixed Price* price type is useful when applying flat rate pricing. The following are scenarios
where flat rate pricing is applicable:

- Store wide clearance sales.
- Configuring pricing for different units of packaging.

Flat sales price for a sub category

> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer 2: ```suggestion
Flat sales price for a subcategory
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

the following:

- Four-pack is $12
- Six-pack is $15
- Twelve-pack is $24

Navigate to :menuselection:`Sales app --> Products --> Pricelist` and select an existing pricelist
or click :guilabel:`New` to create a new one.

On the *Create Pricelist Rules* pop-up window, follow these settings:

- :guilabel:`Apply To`: Select :guilabel:`Product`.
- :guilabel:`Product`: Select *Beer* from the drop-down menu.
- :guilabel:`Price Type`: Select :guilabel:`Fixed Price`.
- :guilabel:`Fixed Price`: Enter `3`.

To set the packaging, select :guilabel:`Pack of 4` from the :guilabel:`Units` drop-down menu. Click
:guilabel:`Save & New` to add the pricelist rule to the pricelist and start a new one.

.. image:: pricing/19-4-example-beer-packaging-fixed-price.png
   :alt: Example of Fixed Price configuration for product packaging type.

For the second pricelist rule, select the same options except for:

- :guilabel:`Fixed Price`: Enter `2.50`.
- The packaging field: Select :guilabel:`Pack of 6` from the drop-down menu.

Click :guilabel:`Save & New` to add the pricelist rule to the pricelist and in the last pricelist

> rule form, repeat the same options expert for:

Reviewer 2: ```suggestion
rule form, repeat the same options except for:
```

   (based on the information from their contact form), that predetermined pricelist can be removed
   from the *Pricelist* field, and the quotation can still be confirmed, and subsequently, turned

>    into a sales order.

Reviewer: This is a touch confusing- I'd rephrase this block so it's more clear.  Aalso, I do not see anything in the pricelist field of a contct after refreshing. Can you confirm this is accurate? I see a grayed out procelist but it is not in black. 

Also, instead of " There is **no way** " I woudl change that to "It is **not** possible to..."
